wasm2map/src/lib.rs

```diff
@@ -341,7 +357,10 @@ impl WASM {
     // wherever possible. So we need to encode the source file data and
     // line, column data for each WASM code segment address in the expected
     // order, so offsets make sense when resolved by the browser (or debugger)
-    fn generate<'a>(&'a self) -> (Vec<String>, Vec<String>) {
+    fn generate<'a>(
+        &'a self,
+        bundle: bool,
+    ) -> (Vec<String>, Vec<String>, Option<Vec<Cow<'static, str>>>) {
```

Comment:
Can you investigate putting this into a struct instead? It's hard to keep track of what information is at what tuple index.

Reply:
Done.
I used the very imaginative name `Generated`, happy to take any advice on that.